policyengine_us/parameters/gov/states/mn/tax/income/deductions/standard/base.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
     href: https://www.revenue.state.mn.us/sites/default/files/2023-01/m1_inst_21.pdf#page=11
   - title: 2022 Minnesota M1 form instructions
     href: https://www.revenue.state.mn.us/sites/default/files/2023-03/m1_inst_22.pdf#page=11
+  - title: 2023 Minnesota M1 form instructions
+    href: https://www.revenue.state.mn.us/sites/default/files/2024-02/m1-inst-23.pdf#page=11
 
 JOINT:
   2021-01-01: 25_050
```

Comment:
I think the page is off here: should be page 3 
Also the amount adjusted, could you double check the page numbers through the PR to make sure it links to the amount?
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/9c11614b-f52b-4dfa-a7e4-b187359ceed3)

Reply:
OK I will correct the pages. The amount will be done later, I just commit in case I lose some previous changes. MN really has a lot to be changed

Reply:
page numbers checked. still have some other problems... let's discuss later